convert-pth-to-ggml.py

```diff
@@ -268,7 +268,25 @@ def main():
             process_and_write_variables(fout, model, ftype, part_id, n_parts)
             del model
 
+<<<<<<< HEAD
```

Comment:
Merge conflict?

Reply:
Right. Fixed in 0a98613d9ee983c691534bbbb58f7d8a899c0494